### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Gianni Cecchetto and Nathan Tschantz

### DISCUSSION THREAD

**Discussion by GianniCecchetto** [heig-vd-dai-course/heig-vd-dai-course#462] [Show and tell]
For this practical work, we thought about creating an Hangman game where 2 players could compete against each other to find the word as fast as possible. The player will be able to see some info about how the other is performing like the number of guesses and the number of life left.

**Comment by GeraudSilvestri** [comment]
It is good for me.

Remember to start simple and add more complex features once the base is working properly

**Comment by GianniCecchetto** [comment]
[Link](https://github.com/GianniCecchetto/dai-practical-work-2-hangman) to our repository

Last commit hash : f1728a703020948a69543cf58462245a62f505de
